screens/StudentInform2.tsx

```diff
@@ -3,18 +3,39 @@ import StyledButton from '../components/StyledButton';
 
 import List from '../components/List';
 import { HomeScreenProps } from './HomeScreen';
-import { useState } from 'react';
+import React, { useState } from 'react';
 import backendFetch from '../lib/backendFetch';
+import StyledAlternativeInput from '../components/StyledAlternativeInput';
 
 const image = require('../assets/images/background.png');
 
 export default function StudentInform2({
   navigation: { navigate },
 }: HomeScreenProps) {
-  const [qualities, setQualities] = useState([
-    { item: 'Example', key: 'Example' },
-  ]);
-  const [hobbies, setHobbies] = useState([{ item: 'Example', key: 'Example' }]);
+  const [qualities, setQualities] = useState('');
+  const [hobbies, setHobbies] = useState('');
+
+  function updateProfile() {
+    const missingFields: string[] = [];
+    console.log('state: ', { qualities, hobbies });
+    //replaceAll doesn't exist, so string.replace with this exact regex works fine
+    if (!qualities.replace(/,/g, '').trim()) {
+      missingFields.push('You have more qualities than this');
+    }
+    if (!hobbies.replace(/,/g, '').trim()) {
+      missingFields.push('Fill in at least 1 hobby');
+    }
+    if (missingFields.length > 0) {
+      alert(missingFields.join('\n'));
+      return;
+    }
+
+    backendFetch('POST', 'account/update', {
+      qualities: JSON.stringify(qualities),
+      hobbies: JSON.stringify(hobbies),
```

Comment:
Is fine but if you commit the code above this has to be changed to
```suggestion
      qualities: qualitiesArr.join(","),
      hobbies: hobbiesArr.join(","),
```